db.js

```diff
@@ -1,16 +1,130 @@
 #!/usr/bin/env node
 /* eslint no-unused-expressions: off */
 
-function create() {
+// set up the connection with db
```

Comment:
I usually start comments with an uppercase letter, but that's just my preference.